### PR TITLE
Feat: styling hero section 

### PR DESCRIPTION
**Pull request by teckno**

### CODE REVIEW

File: src/Pages/Donate.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+
+function DonationPage() {
+  const [amount, setAmount] = useState('1.00');
+
+  useEffect(() => {
+    // Dynamically load PayPal button script
+    const script = document.createElement('script');
+    script.src = 'https://www.paypal.com/sdk/js?client-id=AUCrLayg03De945drbB8zTDdwkGhUrRyPojJUU1u6eLAsJB5yaSOYhdLHenlSyHZLq6Xu5bvserjAkkl';
```
Comment (silasfox) [inline src/Pages/Donate.js:9]: Do you want to save the client-id in plain code on github?

Comment (teckno) [reply src/Pages/Donate.js:9]: Am not so sure if there is a workaround or best practice maybe I just leave it blank and make final changes on the server. I am trying to set up a CI/CD pipeline for my server. Are you well-versed in that area?

**Review by silasfox — APPROVED** [review]

File: src/Pages/Donate.js (hunk 1 above)
Comment (silasfox) [reply src/Pages/Donate.js]: I'm not well versed in that area, but normally, you'd leave some dummy value or leave it empty and have the real value in a configuration file on the server, from which the program reads it at runtime or startup time.

**Review by silasfox — APPROVED** [review]
There's a few typos, but I would just merge this PR and correct them in another PR. Should be no problem.

**Pull request merged**